### PR TITLE
Update Collections governance

### PR DESCRIPTION
**Pull request by MattiSG**
This PR builds upon #111, clarifying some choices that have a strategic impact:

- Define how to join the federation.
- Define how the federation criteria will be assessed.
- Add Sponsor role to collection governance.
- Suggest to inform the community and to check known collections before creating a new collection.

All of the commits are atomic and can be extracted if they prove controversial.

### CODE REVIEW

File: content/collections/create.en.md [hunk 1]
```diff
@@ -5,10 +5,24 @@ weight: 3
...
+
 ## Define metadata
 
 First of all, define the [metadata]({{< relref "collections/metadata" >}}) of the collection you would like to create.
 
+## Check existing collections
+
+Now that you have a clear idea what you would like to track, double-check that there are no [existing federated collections](https://opentermsarchive.org/#collections) that you could contribute to. If you have a doubt about whether some terms you want to track would fit a collection, reach out to the collection maintainers.
+
+If no existing collection could be a good host for the terms you would like to track, then it is relevant to create your own.
```
Comment (clementbiron) [inline content/collections/create.en.md:18]: This explanation about checking existing collections doesn't suggest that creating similar collections is a good thing, although I think it's a good idea to encourage it.

File: content/collections/create.en.md [hunk 2]
```diff
@@ -5,10 +5,24 @@ weight: 3
...
+
+## Inform the community
+
+Starting a new collection is an exciting endeavour, and would strongly benefit from the support of the community who already maintains existing collections. It is strongly recommended to share your intention to create a new collection as early as possible in the process, to get support and identify potential partners.
+
+You can inform the community by posting on the instant messaging system, or [sending an email](mailto:contact@opentermsarchive.org) to the core team.
```
Comment (clementbiron) [inline content/collections/create.en.md:24]: Don't we offer a direct link to Mattermost?

Comment (clementbiron) [inline content/collections/create.en.md:24]:
```suggestion
[Join the community]({{< relref "collections/federation#how-to-join" >}}) to inform the community by posting on the instant messaging system, or [sending an email](mailto:contact@opentermsarchive.org) to the core team.
```

**Review by clementbiron — CHANGES_REQUESTED** [review]
Well done :)

My suggestion about "Check existing collections" seems important to me, the others are not blocking.

File: content/collections/create.en.md (hunk 1 above)
Comment (Ndpnt) [reply content/collections/create.en.md:18]: As maintenance represents a large part of the work and not just the creation of the collection, I think we should encourage firstly contributing to an existing collection to increase the chances of having quality datasets and only secondly, we should encourage creating a collection with similar services, which increase overall resilience.

File: content/collections/create.en.md (hunk 2 above)
Comment (Ndpnt) [reply content/collections/create.en.md:24]: We could explain that if she/he wants to join the instant messaging system, she/he should make a request to us.

**Review by Ndpnt — APPROVED** [review]

**Review by clementbiron — APPROVED** [review]
Having reread it and discussed it together, I think it's perfectly fine as it stands.

**Pull request merged**